stubs/django-environ/environ/environ.pyi

```diff
@@ -0,0 +1,141 @@
+import os
+from collections.abc import Callable, Mapping, MutableMapping
+from logging import Logger
+from typing import IO, Any, ClassVar, SupportsIndex, TypedDict, TypeVar, overload
+from typing_extensions import TypeAlias, Unpack
+from urllib.parse import ParseResult
+
+from .fileaware_mapping import FileAwareMapping
+
+Openable: tuple[type, ...]
+logger: Logger
+
+class NoValue: ...
+
+# Some type aliases to make our life easier
+_Str = str
+_Bytes = bytes
+_Bool = bool
+_Int = int
+_Float = float
+_List = list
+_Tuple = tuple
+_Dict = dict
+
+_T = TypeVar("_T")
+_Cast: TypeAlias = Callable[[_Str], _T]
+_SchemeValue: TypeAlias = _Cast[Any] | tuple[_Cast[Any], Any]
+_BooleanTrueStrings: TypeAlias = tuple[str, ...]
+
+class Env:
+    ENVIRON: MutableMapping[_Str, _Str]
+    NOTSET: ClassVar[NoValue]
+    BOOLEAN_TRUE_STRINGS: ClassVar[_BooleanTrueStrings]
+    URL_CLASS: ClassVar[type[ParseResult]]
+    POSTGRES_FAMILY: ClassVar[_List[_Str]]
+    DEFAULT_DATABASE_ENV: ClassVar[_Str] = "DATABASE_URL"
+    DB_SCHEMES: ClassVar[_Dict[_Str, _Str]]
```

Comment:
no need to use aliases for builtin types, it's more confusing
```suggestion
    ENVIRON: MutableMapping[str, str]
    NOTSET: ClassVar[NoValue]
    BOOLEAN_TRUE_STRINGS: ClassVar[_BooleanTrueStrings]
    URL_CLASS: ClassVar[type[ParseResult]]
    POSTGRES_FAMILY: ClassVar[list[str]]
    DEFAULT_DATABASE_ENV: ClassVar[str] = "DATABASE_URL"
    DB_SCHEMES: ClassVar[dict[str, str]]
```

I didn't change everything, but you get the idea :)

Reply:
This won't work because the `Env` class defines methods `def str(...)`, `def list(...)`, etc. overwriting the builtin types. That was the reason for the aliases. ;) Alternatively, I could explicitly use `builtins.str`, `builtins.list`, etc. everywhere in the `Env` class. What do you think?